IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.m

```diff
@@ -257,24 +257,53 @@ - (BOOL)saveAccount:(nonnull MSIDAccountCacheItem *)account
                                                                                           realm:account.realm
                                                                                            type:account.accountType];
 
-    // Get previous account, so we don't loose any fields
+    // Get previous account, so we don't lose any fields
     MSIDAccountCacheItem *previousAccount = [_dataSource accountWithKey:key serializer:_serializer context:context error:error];
 
     if (previousAccount)
     {
+        // If this item was recently modified by another process (ignoring ourselves), report a *potential* collision
+        [self checkIfRecentlyModifiedAccount:previousAccount context:context];
+
         // Make sure we copy over all the additional fields
         [account updateFieldsFromAccount:previousAccount];
     }
 
     key.username = account.username;
-
+    NSProcessInfo *processInfo = [NSProcessInfo processInfo];
```

Comment:
This is macOS only code in a cross-plat class. It shouldn't be here

Reply:
I moved this to MSIDMacKeychainTokenCache.